api/protos/cloud_controller.proto

```diff
@@ -38,6 +38,7 @@ message Route {
   string guid = 1;
   string host = 2;
   string path = 3;
+  bool internal = 4;
```

Comment:
where does the internal bool’s default live? will it be defaulted? is it optional?

Reply:
In proto3 syntax, explicit defaults are not supported. The implicit default is `false`, which is what we want.